main.go

```diff
@@ -312,8 +313,40 @@ func semVerRelease(release string) string {
 	return strings.Join(releaseMatcher.FindAllString(release, -1), "") + ".0.0"
 }
 
+func rewriteSystemdUnit() (path string, cleanup func(), err error) {
+	original, err := os.ReadFile("minio.service")
+	if err != nil {
+		return
+	}
+
+	fixed := bytes.ReplaceAll(original, []byte("/usr/local"), []byte("/usr"))
```

Comment:
I know this is very ugly. This is typically solved by using templates like `@BINDIR@` in the source file.

The source file unfortunately lives in another repo (https://github.com/minio/minio-service), and that repo has instruction how to install that unit by downloading it directly from the repo (https://github.com/minio/minio-service/blob/24bc1684943aad86eab70a31b1df4350cf16156f/linux-systemd/README.md?plain=1#L35), which will obviously break if we add some sort of templating in there.

We could copy the file here, but maintaining two almost identical copies is ugly as well.